pkg/virt-launcher/notify-client/client.go

```diff
@@ -146,25 +147,46 @@ func libvirtEventCallback(c cli.Connection, domain *api.Domain, libvirtEvent lib
 				client.SendDomainEvent(event)
 				events <- event
 			}
+		} else if interfaceStatus != nil {
```

Comment:
I guess it  is unlikely that this can ever happen before the first two conditions, but shouldn't this be independent?

```go
if interfaceStatus != nil {
[...]
}

if libvirtEvent.Event != nil {
[...]
}

```

Reply:
Yes, unlikely (in the current code), but no harm in it being independent.
Done